client/src/views/Home/JoinParty.jsx

```diff
@@ -76,16 +84,24 @@ function JoinGame({
         >
           Join Game
         </Button>
-      </main>
-      <footer className="app__footer">
         <Button
           onClick={showCreateGameView}
-          type="secondary"
-          className="button-secondary--min-width"
+          className="button-disabled"
+          type="primary"
+          style={{ marginTop: "24px" }}
           icon="+"
         >
           Create Game
         </Button>
+      </main>
+      <footer className="app__footer">
+        <Button onClick={handleHowToPlayModalOpen} className="button-secondary">
+          How To Play
+        </Button>
+        <HowToPlayModal
```

Comment:
Sorry one last thing. Now I wonder if the reason your Modal title was messed up was b/c it was inside of the footer. Since the modal is outside of the flow of the document there is no reason to put it here. 

Please move it to just below the footer (in between `</footer>` and `</>`). I prefer doing that b/c then it is clear that it is not physically inside the footer but exists outside. But also b/c then the css doesn't impact it.  Then check to see if the title has changed and doesn't need the other formatting you applied to it.

Reply:
I think the CSS I applied to the HowToPlayModal affected the title, not the footer. Either way I can take it out of the footer.